DESCRIPTION

```diff
@@ -36,6 +36,7 @@ Imports:
     fst,
     glue,
     here,
+    jsonlite,
```

Comment:
this has been reordered in master, so it's just below.... this is probably unnecessary

Reply:
I tried to merge in master, and these two files are what came through after using `theirs` as  the merge strategy. I can clean these up.